_posts/en/newsletters/2021-04-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+      hours or funds could be lost.
+
+    - *Timelocked outputs:* for most regular onchain payments, a user
+      who wants to CPFP bump can pay for a fee bump using the funds
+      stored in their output from the transaction they want to bump.  In
+      the case of LN, those funds aren't available until the channel
+      close has been fully settled onchain.  That means the user needs
+      to use a separate UTXO to pay the fees.
+
+    To address tho above two concerns, LND is requiring users of anchor
+    outputs to retain at least one confirmed UTXO of reasonable value in
+    their wallet any time a channel is open.  That ensures they can CPFP
+    fee bump when necessary, but it has certain consequences, such as
+    preventing spending the last of your onchain funds (even to open a
+    new channel) while you still have at least one channel open.
+
+    Osuntokun's request is for wallets or services built on LND to let
+    the development team know if any of the above concerns, or any other
+    concerns related to anchor outputs, will cause serious problems.
+    Although the question is specific to LND, the answers may have
+    implications for all LN nodes.
+
+- **Sapio public launch:** Jeremy Rubin [posted][rubin sapio] to the
+  Bitcoin-Dev mailing list an announcement that he has made available
+  the Sapio smart contract development toolkit, a Rust-based library
+  and associated tooling that can be used to create smart contracts
+  expressible using Bitcoin Script.  The language was originally
+  designed to make use of Rubin's proposed
+  [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] opcode
+  (`OP_CTV`) but it can simulate the existence of that opcode, and of
+  other potential features for Bitcoin such as taproot, using a trusted
+  signing oracle.  In addition to the Sapio library, the release also
+  contains extensive documentation and an experimental frontend.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.1rc1][Bitcoin Core 0.21.1] is a release candidate
+  for a version of Bitcoin Core that, if activated, will enforce the
+  rules of the proposed [taproot][topic taproot] soft fork, which uses
+  [schnorr signatures][topic schnorr signatures] and allows use of
+  [tapscript][topic tapscript].  These are, respectively, specified by
+  BIPs [341][BIP341], [340][BIP340], and [342][BIP342].  Also included
+  is the ability to pay [bech32m][topic bech32] addresses specified by
+  [BIP350][], although bitcoins spent to such addresses on mainnet will
+  not be secure until a soft fork such as taproot is activated.  The
```

Comment:
This is a little unclear -- until a soft fork with taproot is activated? until a soft fork that includes taproot is activated?

I can parse that the softfork can be referred to as taproot, but it took me a couple of passes.

Reply:
I'll try to improve, but FYI the soft fork that makes bech32m addresses safe to use doesn't need to be taproot.  E.g., if we abandoned taproot today and instead did something like BIP114 MAST, bech32 would be safe after BIP114 activates when paying to BIP114 addresses.

Reply:
"until taproot is activated (or a similar soft fork defining rules for the new addresses is activated)" perhaps?